WcaOnRails/app/views/persons/_details.html.erb

```diff
@@ -0,0 +1,34 @@
+<div>
+  <div class="text-center">
+    <h2><%= @person.name %></h2>
+    <!-- Handle multiple sub ids. -->
+    <% if @previous_persons.any? %>
+      <h4>
+        (<%= t 'persons.show.previously' %> <%= @previous_persons.map { |person| "#{person.name} - #{person.country.name}" }.join(', ') %>)
+      </h4>
+    <% end %>
+    <% if @person&.user&.avatar %>
+      <%= image_tag @person.user.avatar, class: "avatar" %>
+    <% end %>
+  </div>
+  <div class="table-responsive details">
+    <table class="table table-striped">
+      <thead>
+        <tr>
+          <th><%= t 'common.country' %></th>
+          <th><%= t 'common.user.wca_id' %></th>
+          <%= content_tag :th, t('activerecord.attributes.person.gender') if @person.gender.present? %>
+          <th><%= t 'layouts.navigation.competitions' %></th>
+          <th><%= t 'persons.show.completed_solves' %></th>
+        </tr>
+      </thead>
+      <tbody>
+        <td class="country"><i class="flag f32 <%= @person.country_iso2.downcase %>"></i> <%= @person.country.name %></td>
```

Comment:
nit: It might be nice to create a `flag_icon` helper method like that `cubing_icon` helper method you wrote.